### PR TITLE
Add IN operator to table filter #884

### PR DESCRIPTION
**Pull request by ianberryman**

### CODE REVIEW

File: apps/studio/src/components/tableview/TableTable.vue [hunk 1]
```diff
@@ -553,8 +557,15 @@ export default Vue.extend({
         this.filterMode === FILTER_MODE_BUILDER &&
         this.filter.type && this.filter.field && this.filter.value
       ) {
-        
-        return [this.filter]
+        if (this.filter.type === 'in') {
+          const vals = this.filter.value.split(/,/)
```
Comment (rathboma) [inline apps/studio/src/components/tableview/TableTable.vue:561]: I think this should probably include whitespace either side of the comma - `/\s*,\s*/`

**Review by rathboma — CHANGES_REQUESTED** [review]
looks awesome! I think we just need a single change to the regex and we're good.

**Comment by rathboma** [comment]
@ianberryman looks like Cockroach (and probably redshift) doesn't support the `?` syntax for parameters. I think that's why it was using `$1`, `$2`. See the failed unit tests

File: apps/studio/src/lib/db/clients/utils.ts [hunk 2]
```diff
@@ -81,6 +81,8 @@ export function buildFilterString(filters, columns = []) {
       const column = columns.find((c) => c.columnName === item.field)
       if (column && column.dataType.toUpperCase().includes('BINARY')) {
         return `HEX(${wrapIdentifier(item.field)}) ${item.type} ?`
```
Comment (rathboma) [inline apps/studio/src/lib/db/clients/utils.ts:83]: What if this is both Binary and 'IN'?

**Review by rathboma — CHANGES_REQUESTED** [review]
You are amazing, I have two more super minor changes, then we're good to merge!

File: apps/studio/package.json [hunk 3]
```diff
@@ -19,7 +19,7 @@
     "postinstall": "electron-builder install-app-deps",
     "postuninstall": "electron-builder install-app-deps",
     "release:prs": "./build/get-prs.sh",
-    "test:integration": "TEST_MODE=1 ELECTRON_RUN_AS_NODE=1 yarn electron ../../node_modules/jest/bin/jest.js --config ./jest.integration.config.js"
```
Comment (rathboma) [inline apps/studio/package.json:22]: Not sure why this was deleted? This is needed to have the tests run in the right way

Comment (rathboma) [reply apps/studio/package.json:22]: Oh, I guess this doesn't work on Windows right?

Comment (ianberryman) [reply apps/studio/package.json:22]: I'll fix that, I removed that to try to get tests working locally on Windows

**Pull request merged**